1/0
# 发生异常后，后面对代码不再执行。
print('next')
